agent/ssm/ssm.go

```diff
@@ -45,16 +46,16 @@ func GetSecretsFromSSM(names []string, client SSMClient) (map[string]string, err
 func extractSSMValues(out *ssm.GetParametersOutput) (map[string]string, error) {
 	if out == nil {
 		return nil, errors.New(
-			"fetching secret data from ssm parameter store: empty response")
+			"empty response")
 	}
 
 	if len(out.InvalidParameters) != 0 {
-		errorString := ""
+		var stringValues []string
 		for _, invalid := range out.InvalidParameters {
-			errorString += aws.StringValue(invalid) + ", "
+			stringValues = append(stringValues, aws.StringValue(invalid))
 		}
 		return nil, fmt.Errorf(
-			"fetching secret data from ssm parameter store: invalid parameters: %s", errorString)
+			"invalid parameters: %s", strings.Join(stringValues, ","))
```

Comment:
`invalid ssm parameters`?

Reply:
I called out that it is from SSM parameter store in the place I am using this function in ssmsecret.go. Even it will be used somewhere else later, normally we will add more descriptive messages there. So I think we can keep it like what we have now.